docs/.vitepress/config/sidebar/main.ts

```diff
@@ -10,7 +10,6 @@ export const mainSidebar: DefaultTheme.Sidebar = {
         { text: 'Running Your First Environment', link:'/guide/build-envd'},
         { text: 'Using envd Environment', link:'/guide/ide'},
         { text: 'envd Concepts', link:'/guide/concepts'},
-        { text: 'Modularize your build', link:'/guide/modularize'},
```

Comment:
Is this one hidden?

Reply:
I added it back. It is by accident. Thanks for the review.